qpmodel/PhysicalNode.cs

```diff
@@ -161,7 +161,7 @@ public double InclusiveCost()
             children_.ForEach(x =>
             {
                 if (x is PhysicMemoRef xp)
-                    incCost += xp.Group().minIncCost_;
+                    incCost += xp.Group().nullMinIncCost_;
```

Comment:
better split refactoring code (no or trivial semantic changes, but with rename/function refactoring work) and actual working code.